pleskdistup/common/src/plesk.py

```diff
@@ -237,3 +237,23 @@ def get_from_plesk_database(query: str) -> typing.Optional[typing.List[str]]:
     )
     log.debug(f"Command {cmd} returned {proc.returncode}, stdout: '{proc.stdout}', stderr: '{proc.stderr}'")
     return proc.stdout.splitlines()
+
+
+def get_repository_by_os_from_inf3(inf3_content: typing.Union[ElementTree.Element, str], os: dist.Distro) -> typing.Optional[str]:
+    if isinstance(inf3_content, str):
+        if not inf3_content:
+            return None
+        inf3_content = ElementTree.fromstring(inf3_content)
+
+    for build in inf3_content.findall(".//build"):
```

Comment:
Won't just "//build" work?

Reply:
Unfortunately no. It causes `SyntaxError("cannot use absolute path on element")`